agent/api/task/task.go

```diff
@@ -57,6 +57,7 @@ import (
 	"github.com/containernetworking/cni/libcni"
 	dockercontainer "github.com/docker/docker/api/types/container"
 	"github.com/pkg/errors"
+	"github.com/aws/amazon-ecs-agent/agent/taskresource/envFiles"
```

Comment:
you need to run go fmt to correct the import order.

Reply:
done